src/conn_sock.c

```diff
@@ -13,14 +13,46 @@
 #include <sys/un.h>
 #include <sys/stat.h>
 
-static gboolean attach_cb(int fd, G_GNUC_UNUSED GIOCondition condition, G_GNUC_UNUSED gpointer user_data);
-static gboolean conn_sock_cb(int fd, GIOCondition condition, gpointer user_data);
-static gboolean read_conn_sock(struct conn_sock_s *sock);
-static gboolean terminate_conn_sock(struct conn_sock_s *sock);
-void conn_sock_shutdown(struct conn_sock_s *sock, int how);
-static void sock_try_write_to_masterfd_stdin(struct conn_sock_s *sock);
-static gboolean masterfd_write_cb(G_GNUC_UNUSED int fd, G_GNUC_UNUSED GIOCondition condition, G_GNUC_UNUSED gpointer user_data);
-
+static gboolean attach_cb(int fd, G_GNUC_UNUSED GIOCondition condition, gpointer user_data);
+static gboolean rmt_sock_cb(int fd, GIOCondition condition, gpointer user_data);
```

Comment:
what is rmt and lcl?

Reply:
see conn_sock.h

Reply:
I think I'd prefer `local` and `remote` here. the number of chars isn't that much more